packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -109,6 +137,46 @@ contract ControllerHelper is FlashControllerHelper, IERC721Receiver {
         emit FlashWMint(msg.sender, _vaultId, _wPowerPerpAmount, amountToFlashswap, _collateralAmount);
     }
 
+    /**
+     * @notice flash close position and buy long squeeth
```

Comment:
I think this can be simplified with just 1 flashswap call

- totalToBuy = _wPowerPerpAmount + _minToReceive
- maxToPay = msg.value + vault.collateralAmount - _collateralToWithdraw

do everything in 1 flashswap callback (burn + withdraw collateral)